set.mm

```diff
@@ -645322,6 +645405,65 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
     $( $j usage 'nnmulcom' avoids 'ax-mulcom'; $)
   $}
 
+  ${
+    mvrlsubd.a $e |- ( ph -> B e. CC ) $.
+    mvrlsubd.b $e |- ( ph -> C e. CC ) $.
+    mvrlsubd.1 $e |- ( ph -> A = ( B - C ) ) $.
+    $( Move RHS left subtraction to LHS.  Converse of ~ mvlraddd .
+       (Contributed by SN, 21-Aug-2024.) $)
+    mvrlsubd $p |- ( ph -> ( A + C ) = B ) $=
+      ( caddc co cmin cc subcld eqeltrd addcld pncand eqtrd subcan2d ) ABDHIZCD
+      ABDABCDJIZKGACDEFLMZFNEFARDJIBSABDTFOGPQ $.
+  $}
+
+  ${
+    laddrotrd.a $e |- ( ph -> A e. CC ) $.
+    laddrotrd.b $e |- ( ph -> B e. CC ) $.
+    laddrotrd.1 $e |- ( ph -> ( A + B ) = C ) $.
+    $( Rotate the variables right in an equation with addition on the left,
+       converting it into a subtraction.  Version of ~ mvlladdd with a commuted
+       consequent, and of ~ mvrladdd with a commuted hypothesis.  (Contributed
+       by SN, 21-Aug-2024.) $)
+    laddrotrd $p |- ( ph -> ( C - A ) = B ) $=
+      ( cmin co mvlladdd eqcomd ) ACDBHIABCDEFGJK $.
+  $}
+
+  ${
+    raddcom12d.b $e |- ( ph -> B e. CC ) $.
+    raddcom12d.c $e |- ( ph -> C e. CC ) $.
+    raddcom12d.1 $e |- ( ph -> A = ( B + C ) ) $.
+    $( Swap the first two variables in an equation with addition on the right,
```

Comment:
Same kind of things: it seems strange to "swap" the whole LHS with a part of the RHS.
> Move a right-addition to in the RHS to a subtraction in the LHS, and then use symmetry of equality.

Same for the following results.